tethys_portal/settings.py

```diff
@@ -452,6 +452,9 @@
     "TETHYS_WORKSPACES_ROOT", relative_to_tethys_home("workspaces", as_str=True)
 )
 
+MEDIA_URL = portal_config_settings.pop("MEDIA_URL", "/media/")
+MEDIA_ROOT = portal_config_settings.pop("MEDIA_ROOT", TETHYS_WORKSPACES_ROOT)
```

Comment:
```suggestion
MEDIA_ROOT = portal_config_settings.pop("MEDIA_ROOT", relative_to_tethys_home("media", as_str=True))
```

Should we just have a dedicated default `media` directory? Or does it make sense to have it default to `TETHYS_WORKSPACES_ROOT`?

Reply:
If we do have a separate `media` directory, should we also have a `media` quota?

Reply:
from our meeting on friday I think we decided @sdc50 was going to handle this.